
[Executed at: Wed Mar 14 1:53:01 PDT 2018]

Traceback (most recent call last):
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24781_3/asnlib/submission.py", line 20, in <module>
    imported=getattr(__import__(work_package, fromlist=fun_name), f)
  File "./q5_work.py", line 9
    if n%2!=0 #identify the n is odd
                                   ^
IndentationError: unindent does not match any outer indentation level

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24781_3/asnlib/submission.py", line 22, in <module>
    print('Error: importing '+fun_name+'.')
TypeError: Can't convert 'list' object to str implicitly
Command exited with non-zero status 1
